extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesConfigRecorder.java

```diff
@@ -38,20 +39,13 @@
     // In order to figure out whether the user has actually set the property, we take advantage of the fact that Smallrye
```

Comment:
We may want to drop this comment, since it doesn't make sense anymore.

Reply:
Indeed!